assets/js/admin/wc-product-import.js

```diff
@@ -0,0 +1,78 @@
+/*global ajaxurl, wc_product_import_params */
+;(function ( $, window ) {
+
+	/**
+	 * productImportForm handles the import process.
+	 */
+	var productImportForm = function( $form ) {
+		this.$form = $form;
+		this.xhr   = false;
+		this.mapping = wc_product_import_params.mapping;
+		this.position = 0;
+		this.file = wc_product_import_params.file;
+		this.security = wc_product_import_params.import_nonce;
+
+		// Number of import successes/failures.
+		this.imported = 0;
+		this.failed = 0;
```

Comment:
If we want to do a detailed breakdown of success/failure on the "Done" screen, we'll have to update the PHP ajax handler and save the success/fail info in a temporary option that we delete after importing is finished. This just gives a tally right now.